rosidl_typesupport_cpp/cmake/rosidl_typesupport_cpp_generate_interfaces.cmake

```diff
@@ -102,7 +102,7 @@ target_include_directories(${rosidl_generate_interfaces_TARGET}${_target_suffix}
 if(NOT typesupports MATCHES ";")
   target_include_directories(${rosidl_generate_interfaces_TARGET}${_target_suffix}
     PUBLIC
-    "${CMAKE_CURRENT_BINARY_DIR}/${typesupports}")
+    "$<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/${typesupports}>")
```

Comment:
> should we change this line too? https://github.com/ros2/rosidl_typesupport/blob/master/rosidl_typesupport_cpp/cmake/rosidl_typesupport_cpp_generate_interfaces.cmake#L105

@j-rivero this is the line you referenced?

Reply:
god, need coffee. Yes, did not checkout your branch when doing greps.